features/support/before-hook.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
+const fs = require('fs');
 /* eslint-disable import/no-unresolved */
-const config = require('../../.func_config');
+const config = fs.existsSync('../../.func_config') ? require('../../.func_config') : {};
```

Comment:
`fs.existsSync` is deprecated: https://nodejs.org/api/fs.html#fs_fs_exists_path_callback